src/test/ui/lint/crate_level_only_lint.rs

```diff
@@ -0,0 +1,19 @@
+#![deny(uncommon_codepoints)]
+
+mod foo {
+#![allow(uncommon_codepoints)] //~ ERROR allow(uncommon_codepoints)
```

Comment:
Could you use annotations that make more sense for a human reader?
Full `allow(uncommon_codepoints) can be specified at crate level only` should fit into the line limit in this case.

Reply:
Done.